redis/src/aio/tokio.rs

```diff
@@ -1,5 +1,6 @@
 use super::{AsyncStream, RedisResult, RedisRuntime, SocketAddr};
 use async_trait::async_trait;
+#[allow(unused_imports)] // silence warning in multiple configuration builds
```

Comment:
why is this necessary?

Reply:
A relic from a previous change, will remove it (in my first commit, I also modified Cargo.toml)